l10n_it_delivery_note_base/models/stock_delivery_note_type.py

```diff
@@ -48,6 +48,11 @@ class StockDeliveryNoteType(models.Model):
         "stock.picking.transport.method", string="Method of transport"
     )
 
+    transport_reason_inconsistency_ids = fields.Many2many(
+        "stock.picking.transport.reason",
+        string="Reason of transport (Inconsistency)",
```

Comment:
**suggestion**: Forse una stringa più chiara potrebbe essere `Inconsistent reasons of transport`?

Reply:
aggiornato nome